components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -213,7 +213,7 @@ export const GridItemSponsoredImageClickArea = styled.section<{ otherWidgetsHidd
 export const GridItemNotification = styled('section')`
   position: fixed;
   left: 50%;
-  top: 0;
+  top: 50px;
```

Comment:
Not for this PR but we should look into whether it's feasible to use the [AlertCenter](https://nala.bravesoftware.com/?path=/story/components-alert--alert-center) for our notifications :smile:

Reply:
Agreed - this was just a little band-aid to get rid of the orange colors.